src/datatrove/pipeline/readers/huggingface.py

```diff
@@ -36,13 +35,13 @@ def __init__(
         limit: int = -1,
         skip: int = 0,
         batch_size: int = 1000,
-        progress: bool = False,
+        doc_progress: bool = False,
         adapter: Callable = None,
         text_key: str = "text",
         id_key: str = "id",
         default_metadata: dict = None,
     ):
-        super().__init__(limit, skip, progress, adapter, text_key, id_key, default_metadata)
+        super().__init__(limit, skip, False, doc_progress, adapter, text_key, id_key, default_metadata)
```

Comment:
maybe we should move the progress arguments from `BaseReader` to `BaseDiskReader` so that we don't have to pass False here which doesn't make much sense